// modern/dist/components/Card/styles.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Author = exports.Title = exports.Category = exports.Image = exports.Root = undefined;

var _glamorous = require('glamorous');

var _glamorous2 = _interopRequireDefault(_glamorous);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var Root = exports.Root = _glamorous2.default.article({
  backgroundColor: '#ffffff',
  marginBottom: '60px',
  minWidth: '370px',
  position: 'relative',
  width: '370px'
});

var Image = exports.Image = _glamorous2.default.div({
  backgroundColor: '#000000',
  height: '278px',
  marginBottom: '23px',
  width: '100%'
});

var Category = exports.Category = _glamorous2.default.span({
  display: 'block',
  fontSize: '14px',
  fontWeight: '600',
  letterSpacing: '3px',
  marginBottom: '10px',
  lineHeight: '17px',
  textTransform: 'uppercase'
});

var Title = exports.Title = _glamorous2.default.h1({
  color: '#000000',
  fontSize: '22px',
  fontWeight: '800',
  lineHeight: '29px',
  marginBottom: '10px'
});

var Author = exports.Author = _glamorous2.default.span({
  fontSize: '14px',
  lineHeight: '17px',
  color: '#000000'
});
