// modern/dist/index.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var glamorous = _interopDefault(require('glamorous'));
var React = _interopDefault(require('react'));

var Root = glamorous.button({
  background: '#ffffff',
  borderRadius: '2px',
  boxShadow: '0 0 2px 0 rgba(0,0,0,0.12), 0 2px 2px 0 rgba(0,0,0,0.24)',
  color: '#000000',
  cursor: 'pointer',
  display: 'block',
  fontSize: '14px',
  fontWeight: 'bold',
  textAlign: 'center',
  padding: '12px 20px'
});

var index = (function (_ref) {
  var children = _ref.children;

  return React.createElement(
    Root,
    null,
    children
  );
});

var Root$1 = glamorous.article({
  backgroundColor: '#ffffff',
  marginBottom: '60px',
  minWidth: '370px',
  position: 'relative',
  width: '370px'
});

var Image = glamorous.div({
  backgroundColor: '#000000',
  height: '278px',
  marginBottom: '23px',
  width: '100%'
});

var Category = glamorous.span({
  display: 'block',
  fontSize: '14px',
  fontWeight: '600',
  letterSpacing: '3px',
  marginBottom: '10px',
  lineHeight: '17px',
  textTransform: 'uppercase'
});

var Title = glamorous.h1({
  color: '#000000',
  fontSize: '22px',
  fontWeight: '800',
  lineHeight: '29px',
  marginBottom: '10px'
});

var Author = glamorous.span({
  fontSize: '14px',
  lineHeight: '17px',
  color: '#000000'
});

var index$1 = (function (_ref) {
  var image = _ref.image,
      category = _ref.category,
      title = _ref.title,
      author = _ref.author;

  return React.createElement(
    Root$1,
    null,
    React.createElement(Image, null),
    React.createElement(
      Category,
      null,
      category
    ),
    React.createElement(
      Title,
      null,
      title
    ),
    React.createElement(
      Author,
      null,
      author
    )
  );
});

exports.Button = index;
exports.Card = index$1;
